examples/tv-app/linux/include/application-basic/ApplicationBasicManager.cpp

```diff
@@ -17,6 +17,7 @@
  */
 
 #include "ApplicationBasicManager.h"
+#include "../cluster-helper.cpp"
```

Comment:
most of our other includes are using `<>`. Could we do the same here?

Can we find a different name than "helper"? Why are we including a .cpp file instead of a header file?

Reply:
Renamed now to zcl-strings.h